src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/AuthenticationTests.kt

```diff
@@ -25,17 +24,32 @@ class AuthenticationTests : IntegrationTest()
     @Test
     fun `authentication fails with malformed Auth header`()
     {
-        val result = post(url, auth = GithubTokenHeader("token","bearer"))
+        val result = post(url, auth = GithubTokenHeader("token", "bearer"))
         assertThat(result.statusCode).isEqualTo(401)
         JSONValidator.validateError(result.text,
                 expectedErrorCode = "github-token-invalid",
                 expectedErrorText = "GitHub token not supplied in Authorization header, or GitHub token was invalid")
     }
 
+    @Test
+    fun `authentication fails with invalid github token`()
+    {
+        val result = post(url, auth = GithubTokenHeader("badtoken"))
+        assertThat(result.statusCode).isEqualTo(401)
+        JSONValidator.validateError(result.text,
+                expectedErrorCode = "github-token-invalid",
+                expectedErrorText = "GitHub token not supplied in Authorization header, or GitHub token was invalid")
+    }
+
+
+    @Ignore
     @Test
     fun `authentication succeeds with well-formed Auth header`()
     {
-        val result = post(url, auth = GithubTokenHeader("token"))
+        val token = ""
```

Comment:
This is annoying - I've tested locally with a valid GitHub PAT and it works, but getting a PAT onto the build agents for them to use is not trivial.

Reply:
We were going to create a PAT for one of our robot users weren't we? Did that not work?

Reply:
Making the PAT is fine, it's getting it onto the build agent that's tricky! I.e. do we want to have a PAT persisted in plain text on the build agents? Maybe fine, but wasn't sure. This would be fixed by https://vimc.myjetbrains.com/youtrack/issue/VIMC-2507 - then agents could get the PAT from vault. Maybe we should revisit ^

Reply:
It's probably fine if it's for a fake user who doesn't have any permissions to do anything? Can you make a user in a GitHub org who basically isn't authorized to do anything?